apps/web/src/routes/_content/resources/books/$bookId.tsx

```diff
@@ -76,58 +88,103 @@ function Book() {
           ) : (
             <></>
           )}
-          <Card className="mx-2 md:mx-auto">
-            <div className="my-4 w-full grid-cols-1 grid-rows-1 sm:grid-cols-3 md:grid">
-              <div className="flex flex-col items-center justify-center border-b-4 border-blue-800 md:mr-10 md:border-0">
+          <Card className="md:mx-auto" color="orange">
+            <div className="w-full flex flex-col md:flex-row">
```

Comment:
should add a gap to avoid this issue 
![image](https://github.com/user-attachments/assets/d2d7ef53-4ea8-43df-a7a9-4b8e3ce9364e)